integration_tests/src/debts.rs

```diff
@@ -0,0 +1,353 @@
+use std::collections::HashMap;
+use std::thread;
+use std::time::Duration;
+
+use crate::five_nodes::five_node_config;
+use crate::setup_utils::database::start_postgres;
+use crate::setup_utils::namespaces::*;
+use crate::setup_utils::rita::thread_spawner;
+use crate::utils::{
+    generate_traffic, get_default_settings, get_ip_from_namespace, query_debts, register_to_exit,
+    test_all_internet_connectivity, test_reach_all, test_routes, DEBT_ACCURACY_THRES,
+};
+use althea_types::Identity;
+use log::info;
+use num256::Int256;
+use num_traits::Signed;
+use rita_common::debt_keeper::GetDebtsResult;
+
+pub async fn run_debts_test() {
+    info!("Starting althea debts test");
+
+    /*
+    These are connected as such with their node weights in paranthesis:
+    1(25)---------2(500)
+    |             |
+    |             |
+    |             |
+    |             |
+    |             |
+    3(15)         4(10)---------7(15)
+    | \           |
+    |   \         |
+    |     \       |
+    |       \     |
+    |         \   |
+    |           \ |
+    5(40)         6(20)
+    */
+
+    // Hardcoded debt pairs and optimal routes. 0 represents the internet
+    let pairs: Vec<(u16, u16)> = vec![(7, 0), (3, 0), (7, 2), (5, 1), (1, 4), (2, 5), (7, 5)];
+
+    // Hardcoded optimal paths for each node pair. Validation uses this to test debts for each node in path
+    let mut optimal_routes: HashMap<(u16, u16), Vec<u16>> = HashMap::new();
+    optimal_routes.insert((7, 0), vec![4]);
+    optimal_routes.insert((3, 0), vec![6, 4]);
```

Comment:
this could be pulled out into it's won function like the five node setup.

Reply:
fixed this